src/actions/ActionSource.ts

```diff
@@ -102,6 +102,8 @@ function mergeConstraints(
  * A combat-based action resource in the game (e.g. a free run or free kill).
  */
 export class ActionSource {
+  static defaultPriceFunction = (item: Item) =>
+    mallPrice(item) > 0 ? mallPrice(item) : Infinity;
```

Comment:
This could be worth putting in a util file or something since it'll probably affect another area of code.